bsms.mediawiki

```diff
@@ -280,74 +280,104 @@ wsh(sortedmulti(2,[4b6a9b93/48'/1'/1']tpubDDpNnt2fg3G4ukE8gsifBKank5sGemyMDa6gUj
+** HMAC_KEY (hex): 3d3913f89a1b2dc9e6cb8c70809c2885214be0171fe867a1120860a5c37708be
+** MAC (hex): 1d6eccbd527d07dc0adabf6223deb165bd8ca145ca08619690fdd7d9a1e390d3
+** IV (hex): 1d6eccbd527d07dc0adabf6223deb165
+** CIPHERTEXT (hex): 11dc9c9e1e4f79f97503d3f7db1dce5f83c4b2b4978e550beda9a69ba14c1c2f2ec91b44a99fd13e1bcf89b8d5e9d80c11fb3ea57c9db6fea34dbfe1ca4d014750d589d2d215b03be0d17e3534c7bfca1e733aef4fa1f4a001aa8ff2e861320f7dabcaadb23ec5954f8d4e7b02d80df9bebb28721956232ef3898a29884555d8a258ba21d05aab286b66b485ae5cdd19e004592181691f49c617bbab8feb228047e2463e37827e1bf37805f2609c055278dd55b9f96615a42fad1025a033fb80cea419cf796bd90e861ae143f5b465ac9644899a106ea748c161b90d785d858f5f032f1f2197fbad4ddfc2988ec9f4842a440bd86a312ab81c4878870089782b4c2a3bca7a33972f32eeab
+** <tt>signer_1_key.dat</tt>: <pre>1d6eccbd527d07dc0adabf6223deb165bd8ca145ca08619690fdd7d9a1e390d311dc9c9e1e4f79f97503d3f7db1dce5f83c4b2b4978e550beda9a69ba14c1c2f2ec91b44a99fd13e1bcf89b8d5e9d80c11fb3ea57c9db6fea34dbfe1ca4d014750d589d2d215b03be0d17e3534c7bfca1e733aef4fa1f4a001aa8ff2e861320f7dabcaadb23ec5954f8d4e7b02d80df9bebb28721956232ef3898a29884555d8a258ba21d05aab286b66b485ae5cdd19e004592181691f49c617bbab8feb228047e2463e37827e1bf37805f2609c055278dd55b9f96615a42fad1025a033fb80cea419cf796bd90e861ae143f5b465ac9644899a106ea748c161b90d785d858f5f032f1f2197fbad4ddfc2988ec9f4842a440bd86a312ab81c4878870089782b4c2a3bca7a33972f32eeab</pre>
 
 * Signer 2
-** MASTER_KEY_FINGERPRINT: b0ff5f2b
-** PRIVATE_KEY (m/48’/1’/1’): KwiAywdYNKAyVMno63EwPZv2BGoQaEFmTAaWyqUpQNnxpCJP885a
-** XPUB (m/48’/1’/1’): xpub6CHg6Sbv6WosDgkSGmDFSQJX764ruF9b88BiGPdagcYAG41REGfQZyMHSizZZbcV3MTBdNqkM5SUbT8pxeS4K84JHJN7HoTv8a7vb2CYvxC
+** MASTER_KEY_FINGERPRINT: 66c088c7
+** PRIVATE_KEY (m/48’/1’/1’): L1RPBzJNLh1H1dQ7zevXGDA1RkeVpFABwYz62vth3CKoAkJLBNmT
+** XPUB (m/48’/1’/1’): xpub6D4JY4WYuC61Wykayib6rLTMVCgMEK7iDPxHPNc8Fenappo93aLJRsokhYNVtZDPLrcnSmUMaijRRrANfYYBgz5MeuJDBHSoHwSp3NNojEo
 ** Legacy signature
 ** <tt>signer_2_key.bsms</tt>:
 <pre>BSMS 1.0
-3b7af42cd4cc259d1035557b
-[b0ff5f2b/48'/1'/1']xpub6CHg6Sbv6WosDgkSGmDFSQJX764ruF9b88BiGPdagcYAG41REGfQZyMHSizZZbcV3MTBdNqkM5SUbT8pxeS4K84JHJN7HoTv8a7vb2CYvxC
-Signer 2 key
-H5LDSYqoLjtNnfuff8QYLdbiCQTCuBz4eaxmtijjQbauLiNqMLKSHGYpU/V0w/NCqJTKbXTJvDHMXBYQ8Bc9aQA=</pre>
+318bb031160659658d05bf06
+[66c088c7/48'/1'/1']xpub6D4JY4WYuC61Wykayib6rLTMVCgMEK7iDPxHPNc8Fenappo93aLJRsokhYNVtZDPLrcnSmUMaijRRrANfYYBgz5MeuJDBHSoHwSp3NNojEo
+Signer 1 key
```

Comment:
Signer 2 key

Reply:
fixed in https://github.com/nunchuk-io/bips/commit/8ddce677ea974632007b6fc549ee5ed5d63e5a79